ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -56,6 +56,15 @@ may appear out of chronological order.
 The internals of each entry are organized similar to
 https://keepachangelog.com/en/1.1.0/, adapted to our plan explained above.
 
+## Circa 2022-06-08
+
+## Added
+
+- Added `setBlockForging` to `NodeKernel` which must be used to set / control
+  block forging of the consensus layer.
+- `ProtocolInfo` does not have the `pInfoBlockForging` record field, as a result
```

Comment:
I'd replace

> `ProtocolInfo` does not have the `pInfoBlockForging` record field

with

> We removed the `pInfoBlockForging` record field from the `ProtocolInfo` type

and I'd replace 

> it is no longer polymorphic in the underlying monad.

with

> `ProtocolInfo` no longer needs the monad parameter, so we've removed that too.

Reply:
agreed.